### PR TITLE
Drop favicon.ico in favor of a single icon.png

### LINKED ISSUE #834: Drop favicon.ico in favor of a single icon.png
This is what is done on flutter.io: https://github.com/flutter/website/blob/master/_includes/head.html#L6

```html
<link rel="shortcut icon" href="{{ "/images/favicon.png" | prepend: site.baseurl  }}">
```

### PR DESCRIPTION
**Pull request by chalin**
Fixes #834 

cc @devoncarew

### CODE REVIEW

**Comment by chalin** [comment]
The build is broken, but because of issues with Effective Dart. These will be addressed separately.

File: src/_includes/head.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <title>{% if page.short-title %}{{ page.short-title }}{% else %}{{ page.title }}{% endif %} | {{ site.title }}</title>
 
   <!-- Favicon / Touch Icons -->
-  <link href="/favicon.ico" rel="shortcut icon">
+  <link rel="icon" sizes="64x64" href="/assets/shared/dart/icon/64.png">
```
Comment (devoncarew) [inline src/_includes/head.html]: Shoulld this not still be `rel="shortcut icon"`?

Comment (chalin) [reply src/_includes/head.html]: Thanks for the feedback. From my readings, "shortcut" shouldn't be used anymore.

From [MDN Link types](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types):

> The `shortcut` link type is often seen before `icon`, but this link type is **non-conforming, ignored and web authors must not use it anymore**.

From the [HTML Living Standard — Last Updated 2 May 2018 >> Link types >> `rel=icon`](https://html.spec.whatwg.org/multipage/links.html#rel-icon):

> For historical reasons, the `icon` keyword _may_ be preceded by the keyword "shortcut". ...

If for some reason you know that "shortcut" should be present, please let me know and I'll add it back.

Comment (devoncarew) [reply src/_includes/head.html]: > If for some reason you know that "shortcut" should be present

I don't! I just saw the diff and wanted to make sure it was intentional.

**Pull request merged**